src/ImageSharp/Processing/Extensions/LightnessExtension.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+using SixLabors.ImageSharp.Processing.Processors.Filters;
+using SixLabors.Primitives;
+
+namespace SixLabors.ImageSharp.Processing
+{
+    /// <summary>
+    /// Defines extensions that allow the alteration of the hue component of an <see cref="Image"/>
```

Comment:
Class and value descriptions are copy pasted from hue and will need updating.

Reply:
you right. i copied from hue extension and was not able to change all text in a rush.